pytest_services/locks.py

```diff
@@ -1,12 +1,18 @@
 """Fixtures for supporting a distributed test run."""
 import contextlib
 import errno
-import fcntl
 import json
 import os
 import socket
 import time
 
+try:
+    import fcntl
```

Comment:
please switch to zc.lockfile as you suggested, instead of this

Reply:
umm. [line 74?](https://github.com/pytest-dev/pytest-services/blob/6d5551c09a11ea2bef36bc21924a9581d733aee4/pytest_services/xvfb.py#L74)

Reply:
In [this comment](https://github.com/pytest-dev/pytest-services/issues/23#issuecomment-420361468), I look into using zc.lockfile, but find it has an inherent interface incompatibility with the usage demanded by pytest-services, and moreover, [my attempt to fix that incompatibility](https://github.com/zopefoundation/zc.lockfile/pull/16) is languishing. Therefore, I'd like to revisit this PR as requested.

Reply:
Since I posted that, the zc.lockfile change did finally make it through (after many rounds of trying to get authorized to contribute to the project). I can revisit the zc.lockfile work now.

Reply:
great, thanks